tests/Integration/Behaviour/Features/Scenario/Discount/BO/add_free_gift_discount.feature

```diff
@@ -39,17 +34,3 @@ Feature: Add discount
       | valid_to         | 2019-12-01 00:00:00    |
```

Comment:
We should add an assertion on the type on all the scenarios that don't have it now At least in the BO scenarios, in the FO they are less relevant since that's not what we aim at checking specifically